enterprise/app/trends/trends.tsx

```diff
@@ -213,35 +215,41 @@ export default class TrendsComponent extends React.Component {
               <TrendsChartComponent
                 title="Cache read throughput"
                 data={this.state.lastNDates}
-                extractValue={(date) => +this.state.dateToStatMap.get(date)?.totalDownloadSizeBytes / 1000000}
+                extractValue={(date) =>
+                  bitsPerByte * (+this.state.dateToStatMap.get(date)?.totalDownloadSizeBytes / 1000000)
+                }
                 extractSecondaryValue={(date) =>
-                  +this.state.dateToStatMap.get(date)?.totalDownloadSizeBytes /
-                  +this.state.dateToStatMap.get(date)?.totalDownloadUsec
+                  bitsPerByte *
+                  (+this.state.dateToStatMap.get(date)?.totalDownloadSizeBytes /
+                    +this.state.dateToStatMap.get(date)?.totalDownloadUsec)
                 }
                 extractLabel={this.formatShortDate}
                 formatHoverLabel={this.formatLongDate}
-                formatHoverValue={(value) => (value || 0).toFixed(2) + " MB downloaded"}
-                formatSecondaryHoverValue={(value) => (value || 0).toFixed(2) + " MB / sec"}
-                name="MB downloaded"
-                secondaryName="MB / sec download throughput"
+                formatHoverValue={(value) => (value || 0).toFixed(2) + " Mb downloaded"}
+                formatSecondaryHoverValue={(value) => (value || 0).toFixed(2) + " Mbps"}
+                name="Mb downloaded"
```

Comment:
Can we keep transfer sizes in Megabytes (MB) and only change throughputs to Mbps? Otherwise lgtm

Reply:
Done